app/controllers/users_controller.rb

```diff
@@ -22,9 +22,21 @@ def destroy
 
   def index
     @users = @roster.users
+    @other_users = User.all - @users
     @fallback = @roster.fallback_user
   end
 
+  def transfer
```

Comment:
eh but that's using the word 'roster' more, so I'm not really hooked on changing this.